src/proxy/cache/proxy.js

```diff
@@ -33,17 +33,29 @@ import offlineErrorHelper from 'taoQtiTest/runner/helpers/offlineErrorHelper';
 
 /**
  * The number of items to keep in the cache
- * @type {Number}
+ * @type {number}
+ * @private
  */
-var cacheSize = 20;
+const cacheSize = 20;
 
 /**
  * The number of ms to wait after an item is loaded
  * to start loading the next.
  * This value is more or less the time needed to render an item.
- * @type {Number}
+ * @type {number}
+ * @private
  */
-var loadNextDelay = 450;
+const loadNextDelay = 450;
+
+/**
+ * The default TimeToLive for assets resolving.
+ * Each item comes with a baseUrl that may have a TTL bound to it.
+ * Once this TTL is expired, the assets won't be reachable.
+ * For this reason, we need to remove from the cache items having an expired TTL.
+ * @type {number}
+ * @private
+ */
+const defaultItemTTL = 15 * 60;
```

Comment:
Please comment seconds or milliseconds. It might be obvious here but better to be explicit, since we have a mix of the two in TAO.

Reply:
I add the time unit in the description.